src/App.css

```diff
@@ -28,11 +17,22 @@
   color: #61dafb;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+
+.ReviewCard{
+  display: grid;
+  grid-template-columns: 20vw auto;
+  grid-template-areas:
+  "image title";
+  border: #282c34;
+  border-width: 10vw;
+}
```

Comment:
Can see you've added border details here but it doesn't seem to be appearing - you'll need to add a `border-style` property for it to appear like `border-style: solid` - if you do this however, you may notice that `10vw` is probably too much! Consider lowering that value or using a different unit

Reply:
was aware of it being too much but was confused by why it wasn't showing up! Thanks!